src/components/addForm/addForm.html

```diff
@@ -0,0 +1,22 @@
+<form>
+  <fieldset>
+    <!--label or legend for description?-->
+    <label for="description">DESCRIPTION</label>
+    <input
+      type="text"
+      name="Enter Task"
+      placeholder="Enter Task"
+      id="addTask"
+    />
+  </fieldset>
+  <fieldset>
+    <legend>WHEN</legend>
+    <input type="radio" name="today" id="today" />
+    <label for="today">TODAY</label>
+    <input type="radio" name="tomorrow" id="tomorrow" />
+    <label for="today">TOMORROW</label>
+    <input type="radio" name="choose-date" id="choose-date" />
+    <label for="today">CHOOSE DATE</label>
```

Comment:
The `name` of these radio buttons need to be the same. With radio buttons you are only allowed to select one of the items. However, this only works if the inputs are grouped aka have the same `name`.

All your labels are connected to the first input because of the `for="today"`. Please check how labels and inputs are working together.

Reply:
Arg I had it as the same and the husband said it should be different! 😡 Thank you for your input!